packages/web/src/components/result/ReactiveList.js

```diff
@@ -317,6 +327,20 @@ class ReactiveList extends Component {
 			this.domNode.removeEventListener('scroll', this.scrollHandler);
 		}
 	}
+	// Shape of the object to be returned in onData & renderAllData
+	getAllData = () => {
+		const { size } = this.props;
+		const { currentPage } = this.state;
+		const results = parseHits(this.props.hits) || [];
+		const streamResults = parseHits(this.props.streamHits) || [];
+		return {
+			results,
+			streamResults,
+			loadMore: this.loadMore,
+			base: currentPage * size,
+			triggerClickAnalytics: this.triggerClickAnalytics,
```

Comment:
Please note that we should cover `triggerClickAnalytics` usage in docs properly.

Reply:
ok